src/Type/PostObject/Mutation/PostObjectCreate.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace WPGraphQL\Type\PostObject\Mutation;
+
+use GraphQLRelay\Relay;
+use WPGraphQL\Type\PostObject\PostObjectMutation;
+use WPGraphQL\Types;
+
+/**
+ * Class PostObjectCreate
+ *
+ * @package WPGraphQL\Type\PostObject\Mutation
+ */
+class PostObjectCreate {
+
+	/**
+	 * Holds the mutation field definition
+	 *
+	 * @var array $mutation
+	 */
+	private static $mutation;
+
+	/**
+	 * Defines the create mutation for PostTypeObjects
+	 *
+	 * @param \WP_Post_Type $post_type_object
+	 *
+	 * @return array|mixed
+	 */
+	public static function mutate( \WP_Post_Type $post_type_object ) {
+
+		if ( null === self::$mutation ) {
+			self::$mutation = [];
+		}
+
+		if ( ! empty( $post_type_object->graphql_single_name ) && empty( self::$mutation[ $post_type_object->graphql_single_name ] ) ) :
+
+			/**
+			 * Set the name of the mutation being performed
+			 */
+			$mutation_name = 'create' . ucwords( $post_type_object->graphql_single_name );
```

Comment:
I don't think we want to translate this. . .that would be similar to translating a REST endpoint, which would cause all sorts of issues.